Mathlib/Analysis/CStarAlgebra/CStarMatrix.lean

```diff
@@ -338,7 +338,7 @@ lemma ofMatrix_eq_ofMatrixStarAlgEquiv [Fintype n] [SMul ℂ A] [Semiring A] [St
 
 end basic
 
-variable [Fintype m] [NonUnitalCStarAlgebra A] [PartialOrder A] [StarOrderedRing A]
+variable [Fintype m] [NonUnitalCStarAlgebra A]
```

Comment:
These changes look independent to me (but should be fine as well). Feel free to submit as a separate PR if you don't want to wait.